blueprints/starter/navigation/testing/build.gradle.kts

```diff
@@ -48,7 +48,6 @@ kotlin {
 }
 
 android {
-  namespace = "software.amazon.app.platform.template.navigation"
```

Comment:
Note that the `.testing` suffix was missing here due to copy pasta. Relying on the App Platform Gradle plugin is much nicer.